package.json

```diff
@@ -14,6 +16,7 @@
     "eject": "react-scripts eject"
   },
   "devDependencies": {
+    "eslint-plugin-react-hooks": "^4.6.0",
     "husky": "^8.0.3",
     "prettier": "^2.8.4",
     "pretty-quick": "^3.1.3"
```

Comment:
Do we we have our dependencies int he correct sections? 😃

Reply:
I don't know. I was installing dependencies using npm install. And it was installed automatically. Is something wrong with it?

Reply:
So there is just one issue, eslint should be included in the dev dependencies section.

There are two types of dependencies:
- "dependencies": Packages required by your application in production. (production is just the live website you deply)
- "devDependencies": Packages that are only needed for local development and testing.

Eslint falls into the second one

Reply:
You can read more about this here: https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file

Reply:
But I have it inside devDependencies, no? 
"devDependencies": {
    "eslint-plugin-react-hooks": "^4.6.0",

Maybe I will ask you personally about it :) Thank you!